.taskcluster.yml

```diff
@@ -68,8 +77,9 @@ tasks:
         - go install -tags multiuser -v -ldflags "-X main.revision=%revision%" ./...
         - set CGO_ENABLED=1
         - set GORACE=history_size=7
-        - C:\generic-worker\generic-worker-test-creds.cmd
```

Comment:
What did this do / why is it no longer needed?

Reply:
This was the file that set the credentials, that was burned into the worker image. With this PR, this file is now fetched from taskcluster-secrets.